### PR TITLE
Nasa

### PR DESCRIPTION
**Pull request by aresundnes**

### CODE REVIEW

**Comment by zenangst** [comment]
:space_invader:

File: index.html [hunk 1]
```diff
@@ -26,11 +28,11 @@
...
 
       h1 {
-        font-size: 60px;
+        font-size: 20px;
         margin: 20px 0;
-        background: rgba(255, 255, 255, 1);
-        color: #f57d2d;
-        padding: 30px 20px 20px 20px;
+        color: #fff;
+		padding: 0;
```
Comment (zenangst) [inline index.html:26]: Such indentation

**Comment by mathiasbno** [comment]
:game_die: :spaghetti:

**Comment by mathiasbno** [comment]
@jgorset what is the policy for merging and deploying this?

**Comment by aresundnes** [comment]
I will get @jgorset  to help me out here.. I is github n00b

**Comment by jgorset** [comment]
I was there!

**Comment by jgorset** [comment]
@mathiasbno: What I've been doing so far is just merging things to the `master` branch and making new branches prefixed with `archive/` for old things to keep track of them, but maybe `feature/` would be better (and more consistent with what we do for other projects).

File: index.html (hunk 1 above)
Comment (jgorset) [inline index.html:26]: Much tabs and spaces.

**Pull request merged**